tests/json-fixtures/test_transactions.py

```diff
@@ -136,6 +137,8 @@ def fixture_transaction_class(fixture_data):
     elif fork_name == "Merge":
         # EL fork name is Paris, `ethereum/tests` calls the Network "Merge"
         return ParisTransactionBuilder
+    elif fork_name == "Shanghai":
```

Comment:
Why did you choose to check against a string here and for "Merge" instead of using the `ForkName` enum like Metropolis and earlier?